js-src/graphql/types/Consignments.ts

```diff
@@ -0,0 +1,22 @@
+/* tslint:disable */
```

Comment:
Do we need this line? We're not using tslint on Jenkins at the moment - do you have it set up locally?

Reply:
Copied from an example, removed